site/zenodo_rdm/github/release.py

```diff
@@ -79,4 +79,24 @@ def metadata(self):
             if citation_metadata:
                 output.update(citation_metadata)
 
+        if not output.get("creators"):
```

Comment:
This "pollutes" the release API a bit, I would expect that all the metadata-related operations would be in the metadata class `RDMReleaseMetadata` (or `ZenodoReleaseMetadata`). 

However, that requires that we rework the code and we don't want to do it right now I suppose.